### DISCUSSION TITLE
Enter custom Start and End trim in seconds

### DISCUSSION THREAD

**Discussion by PompJoggie** [mifi/lossless-cut#2056] [General]
Hi there, 

Can you you manually set a custom start and end length to trim?
For example; without even going through the file I know I want to Trim 17 seconds in the beginning and 9 seconds in the end.

**Comment by mifi** [comment]
next version will allow Seek to timecode (<kbd>G</kbd>) to specify a relative time value (e.g. `+17` or `-9`). Then you should be able to achieve this by:
1. <kbd>G</kbd> enter `17`
1. <kbd>I</kbd> (set segment start)
2. Jump to end of video (can be keyboard bound)
3. <kbd>G</kbd> enter `-9`
4. <kbd>O</kbd> (set segment end)
